docs/content/preview/releases/yba-releases/v2.20.md

```diff
@@ -53,17 +53,38 @@ For more information, see [#20827](https://github.com/yugabyte/yugabyte-db/issue
 
 {{< /warning >}}
 
+## v2.20.2.2 - April 1, 2024 {#v2.20.2.2}
+
+**Build:** `2.20.2.2-b1`
+
+**Third-party licenses:** [YugabyteDB](https://downloads.yugabyte.com/releases/2.20.2.1/yugabytedb-2.20.2.2-b1-third-party-licenses.html), [YugabyteDB Anywhere](https://downloads.yugabyte.com/releases/2.20.2.1/yugabytedb-anywhere-2.20.2.2-b1-third-party-licenses.html)
```

Comment:
```suggestion
**Third-party licenses:** [YugabyteDB](https://downloads.yugabyte.com/releases/2.20.2.2/yugabytedb-2.20.2.2-b1-third-party-licenses.html), [YugabyteDB Anywhere](https://downloads.yugabyte.com/releases/2.20.2.2/yugabytedb-anywhere-2.20.2.2-b1-third-party-licenses.html)
```

Reply:
I thought I made this change .. looks like I didn't save properly.. thanks